File: 1ba/PRG.H2503/golang/block01/aufgabe3.go
package main

import "fmt"

func main()	{
	var guthaben float64 = 1000
	var zinsSatz float64 = 3
	var guthabenMitZinsen float64

	guthabenMitZinsen = guthaben * (1 + zinsSatz / 100)
	fmt.Println ( guthabenMitZinsen )
}

File: 1ba/PRG.H2503/block01/aufgabe3.java
public class Aufgabe3 {
    public static void main(String[] args) {
        // Variablen
        double guthaben = 1000;
        int zinsSatz = 3;
        double guthabenMitZinsen;
        // Verarbeitung
        guthabenMitZinsen = guthaben * (1 + zinsSatz / 100);
        // Ausgabe
        System.out.println ( guthabenMitZinsen );
    }
}